cmake-easyinstall

```diff
@@ -17,7 +17,7 @@ fi
 
 # options (undocumented)
 CEI_CMAKE=${CEI_CMAKE:-"cmake"}
-CEI_MAKE=${CEI_MAKE:-"make"}
+CEI_CONFIG=${CEI_CONFIG:-"Release"}
```

Comment:
You should add that to the configure call `-DCMAKE_BUILD_TYPE=...`

Reply:
Also, I like `RelWithDebInfo` as default config, too.
(And Spack does so as well.)